### PR TITLE
Use v2 API on application page

### PR DESCRIPTION
**Pull request by luontola**
#852

### CODE REVIEW

File: src/cljs/rems/actions/request_decision.cljs [hunk 1]
```diff
@@ -11,26 +10,32 @@
...
+   {:db (assoc db
+               ::comment ""
+               ::potential-deciders #{}
+               ::selected-deciders #{})
+    ::fetch-potential-deciders [(get-in db [:identity :user])
+                                #(rf/dispatch [::set-potential-deciders %])]}))
 
 (rf/reg-sub ::potential-deciders (fn [db _] (::potential-deciders db)))
+(rf/reg-event-db
+ ::set-potential-deciders
```
Comment (opqdonut) [inline src/cljs/rems/actions/request_decision.cljs]: was this event really missing previously?

File: src/cljs/rems/application.cljs [hunk 2]
```diff
@@ -74,26 +57,23 @@
...
-
-
-
-
-
-
-
-
-
+            (for [{:keys [type field-id]} (filter :field-id errors)]
```
Comment (opqdonut) [inline src/cljs/rems/application.cljs]: or perhaps
```
(for [{:keys [type field-id]} errors
      :when field-id]
   ...)

Comment (opqdonut) [reply src/cljs/rems/application.cljs]: or perhaps factor out a `format-validation-error` function that can handle field and license errors?

File: src/cljs/rems/actions/request_decision.cljs (hunk 1 above)
Comment (luontola) [reply src/cljs/rems/actions/request_decision.cljs]: It was accidentally removed in commit 6cac8dd0c3de8e2da0860674f7c38c4492df111e

**Pull request merged**